### PR TITLE
feat: we now properly parse complex and nested expressions

### PR DESCRIPTION
**Pull request by nlunets**
Some known issue around ordering of mixed element set that would be too annoying to fix right now are also known :)

### CODE REVIEW

File: packages/edmx-parser/utils/edmx.d.ts [hunk 1]
```diff
@@ -345,14 +346,28 @@ declare namespace EDMX {
     export type UrlRefWrapper = Record<'UrlRef', URL>;
     export type RecordWrapper = Record<'Record', RecordExpression>;
     export type CollectionWrapper = Record<'Collection', CollectionExpression>;
-    export type ApplyWrapper = Record<'Apply', ApplyExpression>;
+    export type ApplyWrapper = {
+        Apply: ApplyExpression;
+        Function: ApplyExpression;
+    };
+    export type AndWrapper = Record<'And', ApplyExpression>;
```
Comment (drktfl) [inline packages/edmx-parser/utils/edmx.d.ts:353]: hmm, maybe these make sense, maybe not, but why is everything either an Apply or an If?

**Comment by nlunets** [comment]
I'll accept that coverage level

File: packages/edmx-parser/utils/edmx.d.ts (hunk 1 above)
Comment (nlunets) [reply packages/edmx-parser/utils/edmx.d.ts:353]: They are just `any` in the end.
Not perfect but not improtant

Comment (drktfl) [reply packages/edmx-parser/utils/edmx.d.ts:353]: ok

**Review by drktfl — APPROVED** [review]

**Pull request merged**